test/integration/basic_test.go

```diff
@@ -28,8 +120,36 @@ func TestBasicPFCPAssociation(t *testing.T) {
 	err := pfcpClient.SetupAssociation()
 	require.NoErrorf(t, err, "failed to setup PFCP association")
 
-	time.Sleep(time.Second*10)
+	time.Sleep(time.Second * 10)
```

Comment:
Why so long?

Reply:
We wait here for a few heartbeats and then check.